src/components/Orders/OrdersContainer.js

```diff
@@ -0,0 +1,117 @@
+                                    <Box
+                                        key={index}
+                                        p={3}
+                                        mb={3}
+                                        spacing={10}
+                                        borderWidth='1px'
+                                        borderRadius='16px'
+                                    >
+                                        <SimpleGrid columns={3} spacing={10}>
+                                            <Box height='80px'>
+                                                <Text color='gray'>
+                                                    Order placed
+                                                </Text>
+                                                {/* order.created_at.getMonth(), order.created_at.getDate(), order.created_at.getFullYear() */}
+                                                {/* /{order.created_at.getDate()}/{order.created_at.getFullYear()} */}
+                                                <Text>
+                                                    {new Date(
+                                                        order.created_at
+                                                    ).toLocaleDateString(
+                                                        {},
+                                                        {
+                                                            timeZone: 'UTC',
+                                                            month: 'short',
+                                                            day: '2-digit',
+                                                            year: 'numeric',
+                                                        }
+                                                    )}
+                                                </Text>
+                                            </Box>
+                                            <Box height='80px'>
+                                                <Text color='gray'>Total</Text>
+                                                <Text>{`$${order.totalAmount}`}</Text>
+                                            </Box>
+                                            <Box height='80px'>
+                                                <Text color='gray'>
+                                                    Order #
+                                                </Text>
+                                                <Text>{order._id}</Text>
+                                            </Box>
+                                        </SimpleGrid>
+                                        <Divider />
+                                        {order.productsPurchased.map(
+                                            (product, index) => {
+                                                return (
+                                                    <div key={index}>
```

Comment:
You could have used `Box` for consistency with the rest of the app but this is fine.